src/eva/vision/data/transforms/common/resize_and_clamp.py

```diff
@@ -4,6 +4,7 @@
 
 import torch
 import torchvision.transforms.v2 as torch_transforms
+from torchvision.transforms import v2
```

Comment:
duplication here, already all the transforms are in `v2` but just renamed as `torch_transforms` -- lets keep one of them

Reply:
removed `torch_transforms`